src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+}
+
+body {
+  margin: 0;
+  font-family: "Avenir", sans-serif;
+}
+
+.bigSearch {
+  margin: 20px 72px 20px 8px;
+}
+
+.smallSearch {
+  margin: 20px 43px 10px 8px;
+}
+
+input {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+}
+
+.input--big {
+  height: 66px;
+  padding-left: 62px;
+}
+
+.input--small {
+  height: 38px;
+  padding-left: 33px;
+}
+
+input:hover {
+  border-color: rgb(142, 64, 64);
+}
+
+input:focus {
+  border-color: rgb(23, 230, 23);
+  font-family: "Avenir", sans-serif;
+  font-style: bold;
+}
+
+.input--big::placeholder {
+  color: #3d4e61;
+  font-size: 16px;
+  font-style: normal;
+  line-height: 22px;
+  font-weight: 300;
+}
+
+.input--small::placeholder {
+  color: #3d4e61;
+  font-size: 14px;
+  font-style: normal;
+  line-height: 19px;
+  font-weight: 300;
+}
+
+.icon {
+  position: absolute;
+  top: 45px;
+  left: 34px;

```

Comment:
lets just set the padding for the input, no need to set each element

Reply:
I didn't understand this moment.